### ISSUE TITLE
🔗 🐞 Bug Report: Links in "Loop Carousel Widget" not working in Header with Mega-Menu (Duplicate of #23705) - [ED-11434]

### ISSUE THREAD

**Issue by tobias992** [elementor/elementor#23810]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I created a loop template for the actual posts and put it in my header as "Loop Carousel Widget". The problem now is that the links to the post pages do not work. They only work after a first left/right swipe in the loop widget.

### Steps to reproduce

1. Create loop template with links (e.g. a button to post page)
2. Use Mega-Menu and use "Loop Carousel Widget" to list posts
3. Then you see that links are not working

After a left/right swipe in "Loop Carousel Widget" or a first auto scroll (if active) the links are working.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: nginx
	MySQL version: mariadb.org binary distribution v10.11.4-MariaDB-1:10.11.4+maria~deb11
	PHP Version: 8.0.29
	PHP Memory Limit: 128M
	PHP Max Input Vars: 4000
	PHP Max Post Size: 255M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.3.1
	Site URL: https://mydomain.com
	Home URL: https://mydomain.com
	WP Multisite: No
	Max Upload Size: 250 MB
	Memory limit: 512M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: de-DE
	Timezone: Europe/Berlin
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 2.0.0
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.8.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: de_DE
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.6 Safari/605.1.15

== Active Plugins ==
	Advanced Custom Fields PRO
		Version: 6.2.1.1
		Author: WP Engine

	Elementor
		Version: 3.16.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.16.1
		Author: Elementor.com

	Embed Privacy
		Version: 1.8.0
		Author: Epiphyt

	FileBird Pro
		Version: 5.5.2
		Author: Ninja Team

	Resize Image After Upload
		Version: 1.8.6
		Author: ShortPixel

	WP-Sweep
		Version: 1.1.8
		Author: Lester 'GaMerZ' Chan


== Eigenschaften ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor-Experimente ==
	Optimierte DOM Ausgabe: Aktiv
	Verbessertes Laden von Assets: Aktiv
	Verbessertes Laden von CSS: Inaktiv
	Inline-Schriftarten-Symbole: Aktiv
	Zusätzliche benutzerdefinierte Breakpoints: Inaktiv
	admin_menu_rearrangement: Standardmäßig deaktiviert
	Flexbox Container: Aktiv
	Upgrade Swiper Library: Inaktiv
	Grid Container: Inaktiv
	Default to New Theme Builder: Inaktiv
	Hello Theme Header & Footer: Inaktiv
	Editor Top Bar: Inaktiv
	Startseiten: Inaktiv
	Nested Elements: Aktiv
	Hintergrundbilder nachladen: Inaktiv
	Global Style Guide: Inaktiv
	Page Transitions: Inaktiv
	Notes: Inaktiv
	Form Submissions: Aktiv
	Scroll Snap: Inaktiv
	Menü: Aktiv
	Taxonomy Filter: Aktiv


== Protokoll ==
	


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Kompatibel

== Elementor Pro - Compatibility Tag ==
	

				

````

</details>

**Comment by nicholaszein** [comment]
Hello @tobias992!

Thank you for your submission! 🙏 

### 🔗 🐞 This issue is a Duplicate of #23705 

Which is on the same subject. 

For this reason, I'm closing this thread and 🔃 **Merging** it with the original, so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards

**Comment by nicholaszein** [comment]
Hello there @tobias992!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor 3.17.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
